packages/components/src/theme/grid.scss

```diff
@@ -67,6 +67,10 @@
   font-weight: bold;
 }
 
+.phi-protected {
```

Comment:
Are you planning on having a follow-up PR for bumping `@labkey/components` and `@labkey/theme` in the core module? Or are you going to do that in tandem with Kevin's PR?

Reply:
Will be figuring that out today, I think I'm going to do that as part of Kevin's PR, but it wouldn't be a big deal if it was a separate PR.